grails-app/conf/Config.groovy

```diff
@@ -497,7 +497,13 @@ openboxes {
                             state6 = ["Expiration(s)"]
                         }
                         labels {
-                            state5 = ["today", "within 30 days", "within 90 days", "within 180 days", "within 360 days"]
+                            state5 = [
+                                [code : "react.dashboard.timeline.today.label", message : "today"],
+                                [code : "react.dashboard.timeline.within30Days.label", message : "within 30 days"],
```

Comment:
@julienreliefapps @awalkowiak @DominikaHatala At some point we should consider refactoring translation codes like this to a more generic version and pass arguments ...
```
[code : "react.default.withinXDays.label", args: ['30'], message : "within 30 days"],
[code : "react.default.withinXDays.label", args: ['90'], message : "within 90 days"],

```

Reply:
@jmiranda  yes good idea ! @awalkowiak , @DominikaHatala does the translation passing argument is already working in the react side ?

Reply:
@jmiranda Yeah this is the part of these translations I disliked the most, but I somehow forgot about the arguments in translations.

Reply:
@jmiranda @julienreliefapps there is an option to add arguments in translations on the react side, but there's a catch:
https://ryandrewjohnson.github.io/react-localize-redux-docs/#dynamic-translations
There might be one tiny issue with it. The placeholder for argument in translation in this react library is `"${argumentName1}${argumentName2}"`, into which arguments are passed as an object with argument names, and for grails it is `"{0}{1}{2}"` with arguments passed in as an array. If we want to do this for react side, then we won't be able to use `react.` properties on the grails side (if we don't need it there, then I think we're good).

Reply:
Ok I see, I am gonna check that @jmiranda @awalkowiak